### PR TITLE
main file is index.html

### PR DESCRIPTION
**Pull request by keertip**
fix for #48 

@sethladd

### CODE REVIEW

**Comment by sethladd** [comment]
Coolness, thanks!

File: lib/src/generator.dart [hunk 1]
```diff
@@ -51,48 +51,46 @@ body {
...
+
+    File f = joinFile(new Directory(out.path), [fileName]);
+    htmlFiles.add(fileName);
+    print('generating ${f.path}');
+    html.start(
+        title: 'Package ${packageName}',
+        cssRef: css.cssHeader,
+        theme: css.theme,
+        inlineStyle: bootstrapOverrides);
+    html.generateHeader();
```
Comment (sethladd) [inline lib/src/generator.dart]: method cascades?

File: lib/src/generator.dart [hunk 2]
```diff
@@ -51,48 +51,46 @@ body {
...
-      html.endTag();
-      html.endTag(); // div.container
-      html.generateFooter();
-      html.end();
-      f.writeAsStringSync(html.toString());
+    var fileName = 'index.html';
+
+    File f = joinFile(new Directory(out.path), [fileName]);
+    htmlFiles.add(fileName);
+    print('generating ${f.path}');
```
Comment (sethladd) [inline lib/src/generator.dart]: future note: consider a logging framework to control log verbosity?

Comment (keertip) [inline lib/src/generator.dart]: Yes, in the works. Makes for easier testing too.

Comment (sethladd) [inline lib/src/generator.dart]: Nice!

**Pull request merged**